deals/module/deals.go

```diff
@@ -331,7 +332,7 @@ func (m *Module) Watch(ctx context.Context, proposals []cid.Cid) (<-chan deals.S
 			select {
 			case <-ctx.Done():
 				return
-			case <-time.After(util.AvgBlockTime):
+			case <-time.After(m.pollDuration):
```

Comment:
When watching deals, use the configured polling duration instead of a fixed `AvgBlockTime` (30s). Despite this will give a very fast status updating, considering sealing takes hours doesn't make sense to be so demanding.
Now the default is 5 min, which actually is quite low too... but now can be easily changed via flags/envs.